connectivity/netsocket/include/netsocket/InternetDatagramSocket.h

```diff
@@ -74,6 +75,58 @@ class InternetDatagramSocket : public InternetSocket {
     nsapi_size_or_error_t recvfrom(SocketAddress *address,
                                    void *data, nsapi_size_t size) override;
 
+    /** Send datagram and ancillary data to the specified address.
+     *
+     *  By default, sendto blocks until data is sent. If socket is set to
+     *  nonblocking or times out, NSAPI_ERROR_WOULD_BLOCK is returned
+     *  immediately.
+     *
+     *  It uses sendmsg with zero ancillary data
```

Comment:
This is the documentation for send msg, please amend.

Reply:
Done